DeepSpeech.py

```diff
@@ -402,7 +406,7 @@ def BiRNN(batch_x, seq_length, dropout, reuse=False, batch_size=None, n_steps=-1
     # This is done to prepare the batch for input into the first layer which expects a tensor of rank `2`.
 
     # Permute n_steps and batch_size
-    batch_x = tf.transpose(batch_x, [1, 0, 2])
+    batch_x = tf.transpose(batch_x, [1, 0, 2, 3])
     # Reshape to prepare input for first layer
     batch_x = tf.reshape(batch_x, [-1, n_input + 2*n_input*n_context]) # (n_steps*batch_size, n_input + 2*n_input*n_context)
```

Comment:
Shouldn't your additional dimension in `batch_x = tf.transpose(batch_x, [1, 0, 2, 3])` somehow occur in this comment?

Reply:
The comment is referring to the shape *after* the reshape, which is not changed.